### PR TITLE
Logsinjestion SDK initial commit

### PR DESCRIPTION
**Pull request by rakshith91**

### CODE REVIEW

File: sdk/monitor/azure-monitor-ingestion/MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1,6 @@
+recursive-include tests *.py *.yaml
```
Comment (rakshith91) [inline sdk/monitor/azure-monitor-ingestion/MANIFEST.in:1]:
```suggestion
recursive-include tests *.py
```

File: sdk/monitor/azure-monitor-ingestion/azure/monitor/ingestion/_helpers.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    for log in logs:
+        # each char is 4 bytes
+        size = len(json.dumps(log)) * CHAR_SIZE
+        if chunk_size + size < MAX_CHUNK_SIZE:
+            curr_chunk.append(log)
+            chunk_size += size
+        else:
+            chunks.append(curr_chunk)
+            curr_chunk = [log]
+            chunk_size = 0
```
Comment (srnagar) [inline sdk/monitor/azure-monitor-ingestion/azure/monitor/ingestion/_helpers.py:26]: This should be set to the current log's size 
```suggestion
            chunk_size = size
```

File: eng/.docsettings.yml [hunk 3]
```diff
@@ -88,6 +88,7 @@ known_content_issues:
   - ['sdk/storage/README.md', '#4554']
   - ['sdk/textanalytics/azure-ai-textanalytics/samples/README.md', '#4554']
   - ['sdk/monitor/azure-monitor-opentelemetry-exporter/README.md', '#4554']
+  - ['sdk/monitor/azure-monitor-ingestion/README.md', '#4554']
```
Comment (rakshith91) [inline eng/.docsettings.yml:91]:
```suggestion
```

[the remark this answers is not in the document]
Comment (rakshith91) [reply eng/.docsettings.yml]: we don't need this anymore - removed it :)

**Review by srnagar — APPROVED** [review]
LGTM!

File: sdk/monitor/azure-monitor-ingestion/CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Release History
+
+## 1.0.0b1 (Unreleased)
```
Comment (rakshith91) [inline sdk/monitor/azure-monitor-ingestion/CHANGELOG.md:3]:
```suggestion
## 1.0.0b1 (2022-07-13)
```

**Pull request merged**